ui/src/login/ValidatedSignInForm.js

```diff
@@ -44,7 +44,7 @@ export default class ValidatedSignInForm extends Component {
       if (response.status === 200) this.props.history.push('/landing')
       else {
         this.setState({
-          invalidFormErrorMsg: response.message,
+          invalidFormErrorMsg: "I'm sorry your credentials is not valid. Please try again",
```

Comment:
Should be "are not valid", not "is not valid".  A designer may have an opinion on what this messaging should be. We have to keep in mind that majority of this experience is done by a kid. 

Also, catch your fork up with master to skip the GAE deploy check.

Reply:
Got it. I will make that change. Just want to confirm that a kid/kids will be the end user of this product and they have to understand the invalidFormerrorMsg

Reply:
@hellonoosh what do you want to do here?